### PR TITLE
docs: add how to run s3 locally and troubleshooting docs

### PR DESCRIPTION
**Pull request by gidjin**
## Proposed changes

As discussed in slack, documenting how we toggle between S3 and local file storage for local development. Also added a troubleshooting guide with an error we have all run into the last couple weeks.

@jbecker01 & @shkeating added you for review of the troubleshooting steps, but if you want to check out the S3 doc that's fine too :)

### CODE REVIEW

**Review by shkeating — APPROVED** [review]

File: docs/how-to/use-s3-locally.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Use dev AWS S3 locally
+
+### Inspect the `.envrc`
+
+The `.envrc` file that is checked into the repo will contain the following:
+
+```sh
+export S3_BUCKET_NAME='test_bucket'
+export S3_REGION='test_region'
```
Comment (minhlai) [inline docs/how-to/use-s3-locally.md:23]: I noticed that in the Personnel API, our environment variable names for S3 are different. Should we standardize that in the Personnel API or change the doc to account for both names?

```
export AWS_DEFAULT_REGION='test_region'
export AWS_ACCESS_KEY_ID='thiscanbeanything'
export AWS_SECRET_ACCESS_KEY='thiscanbeanything'
```

Comment (gidjin) [reply docs/how-to/use-s3-locally.md:23]: Good catch, probably we can make them match. I'll check in my PR for https://github.com/USSF-ORBIT/ussf-personnel-api/pull/12 and if it works I'll merge this in. If not I'll update the docs.

Comment (gidjin) [reply docs/how-to/use-s3-locally.md:23]: I think standardizing would be good. Where are you seeing the discrepancy? The `.envrc` for the cms & personnel api have the same var names, is it in AWS that things don't match?

``` 
export S3_BUCKET_NAME='test_bucket'
export S3_REGION='test_region'
export S3_ACCESS_KEY_ID='thiscanbeanything'
export S3_SECRET_ACCESS_KEY='thiscanbeanything'
```

Comment (gidjin) [reply docs/how-to/use-s3-locally.md:23]: I plan to update them to match so I'm going to merge this pr.

**Pull request merged**